doc/usage/configuration.rst

```diff
@@ -2043,17 +2043,48 @@ information.
            ``babel``, not ``polyglossia``.
 
      ``'fontpkg'``
-        Font package inclusion, default ``'\\usepackage{times}'`` (which uses
-        Times for text, Helvetica for sans serif and Courier for code-blocks).
+        Font package inclusion, the default is ``'\\usepackage{times}'`` which
+        uses Times for text, Helvetica for sans serif and Courier for monospace.
 
         .. versionchanged:: 1.2
            Defaults to ``''`` when the :confval:`language` uses the Cyrillic
            script.
-        .. versionchanged:: 1.5
-           Defaults to ``''`` when :confval:`latex_engine` is ``'xelatex'``.
-        .. versionchanged:: 1.6
-           Defaults to ``''`` also with ``'lualatex'``.
-
+        .. versionchanged:: 2.0
+           Support for individual Greek and Cyrillic letters:
+
+           - In order to support occasional Cyrillic (физика частиц)
+             or Greek letters (Σωματιδιακή φυσική) in
+             a document whose language is English or a  Latin European
+             one, the default set-up is enhanced (only for ``'pdflatex'``
+             engine) to do:
+
+             .. code-block:: latex
+
+                \substitutefont{LGR}{\rmdefault}{artemisia}
+                \substitutefont{LGR}{\sfdefault}{neohellenic}
+                \substitutefont{LGR}{\ttdefault}{cmtt}
+                \substitutefont{T2A}{\rmdefault}{fcm}
+                \substitutefont{T2A}{\sfdefault}{fcs}
+                \substitutefont{T2A}{\ttdefault}{fct}
+
+             For this however, the ``'fontenc'`` key must be used to tell
+             LaTeX to load the ``LGR`` (Greek) or ``T2A`` (partial Cyrillic)
+             font encoding. If ``'fontenc'`` is not modified the above lines
+             are not executed.
```

Comment:
as explained in previous comment, in earlier version of this PR, I had to the contrary automatically added LGR and T2A to `fontenc` loading. But to support this fully with the above default set-up this meant requiring Greek and Cyrillic fonts for all Sphinx projects with ``'pdflatex'`` engine, even those not having usage for Greek and Cyrillic Unicode letters which is almost the totality of existing projects: Cyrillic was not supported (except of course for Cyrillic documents), and Greek was via escaping to math mark-up but I think few people must have used this.